### PR TITLE
feat(Datasets): add help links

### PR DESCRIPTION
**Pull request by cheikhgwane**
Add contextual help to datasets
## Changes

Please list / describe the changes in the codebase for the reviewer(s).

- Add links to datasets wiki.

## Screenshots / screencast

If relevant, please add some screenshots or a short video.

[Screencast from 10-18-2023 10:54:39 AM.webm](https://github.com/BLSQ/openhexa-frontend/assets/25453621/f782bb0b-4145-4db6-93ca-eb48ec2388ef)

### CODE REVIEW

File: src/pages/workspaces/[workspaceSlug]/datasets/[datasetSlug].tsx [hunk 1]
```diff
@@ -84,7 +84,19 @@ const WorkspaceDatasetPage: NextPageWithLayout = (props: Props) => {
...
-      <WorkspaceLayout workspace={workspace}>
+      <WorkspaceLayout
+        workspace={workspace}
+        helpLinks={[
+          {
+            label: t("About datasets"),
+            href: "https://github.com/BLSQ/openhexa/wiki/User-manual#datasets",
+          },
+          {
+            label: t("Using the workspace datasets"),
```
Comment (pvanliefland) [inline src/pages/workspaces/[workspaceSlug]/datasets/[datasetSlug].tsx:95]: Using the OpenHexa SDK with datasets?

**Review by pvanliefland — CHANGES_REQUESTED** [review]
Nice, I just have a suggestion for the label (SDK)

File: src/pages/workspaces/[workspaceSlug]/datasets/[datasetSlug].tsx (hunk 1 above)
Comment (cheikhgwane) [reply src/pages/workspaces/[workspaceSlug]/datasets/[datasetSlug].tsx:95]: Cool !

**Review by pvanliefland — APPROVED** [review]
LGTM!

**Pull request merged**